hummingbot/client/ui/completer.py

```diff
@@ -44,8 +44,9 @@ def __init__(self, hummingbot_application):
         self._export_completer = WordCompleter(["keys", "trades"], ignore_case=True)
         self._balance_completer = WordCompleter(["limit", "paper"], ignore_case=True)
         self._history_completer = WordCompleter(["--days", "--verbose", "--precision"], ignore_case=True)
-        self._gateway_completer = WordCompleter(["create", "config", "connect", "generate-certs", "status", "test-connection", "start", "stop"], ignore_case=True)
+        self._gateway_completer = WordCompleter(["create", "config", "connect", "connector-tokens", "generate-certs", "status", "test-connection", "start", "stop"], ignore_case=True)
         self._gateway_connect_completer = WordCompleter(GATEWAY_CONNECTORS, ignore_case=True)
+        self._gateway_connector_tokens_completer = WordCompleter(sorted(AllConnectorSettings.get_gateway_evm_amm_connector_names()), ignore_case=True)
```

Comment:
```suggestion
        self._gateway_connector_tokens_completer = WordCompleter(sorted(AllConnectorSettings.uses_gateway_generic_connector()), ignore_case=True)
```

This is a more flexible alternative.